src/documents/tutorial-prefsFrameworkMinimalFootprint/MinimalFootprint.md

```diff
@@ -0,0 +1,133 @@
+* Use the grade version of a full page [Preferences Editor](../PreferencesEditor.md)
+* Lazy load
+
+### Minified Custom Build ###
+
+Infusion provides a build tool for creating minified custom packages. This is essential for creating the smallest package which requires the fewest requests. From the Infusion root directory simply run `grunt custom --include="preferences"`, which will create a zip file containing all of the bundled code.
+
+See: "[How Do I Create an Infusion Package](https://github.com/fluid-project/infusion/blob/master/README.md#how-do-i-create-an-infusion-package)" documentation for the full list of steps.
+
+### Full Page Preferences Editor ###
+
+Most website integrations of a [Preferences Editor](../PreferencesEditor.md) make use of [UI Options](../tutorial-userInterfaceOptions/UserInterfaceOptions.md). UI Options provides a convenient interaction for users to quickly review and modify their preferences without having to leave the page, and presents a live preview of their adjustments. However, this also adds additional overhead to instantiate the preferences editor and load in all of the required resources. A Full Page preference editor, on the other hand, is designed to run as a stand alone page and can be configured with a preview window. Be sure to provide the user with a link to the full page preferences editor, for example a link on each content page or in the site's user settings.
+
+<div class="infusion-docs-note"><strong>Note:</strong> Preferences Editors can be configured either through an <a href="../AuxiliarySchemaForPreferencesFramework.md">Auxiliary Schema</a> or directly through grades. The grade version is slightly more complicated to configure, but removes computation time for processing the schema.</div>
+
+#### Instantiating a Full Page Preferences Editor ####
+
+The following example instantiates a Full Page Preferences Editor with a preview window.
+
+```javascript
+    /*
+     * The various starter gradeNames mentioned below indicate that the
+     * "starter" adjusters and enactors should be used. These correspond to the
+     * same set of adjusters and enactors used in a typical UI Options
+     * configuration. However, a custom set of adjusters and enactors could be
+     * configured here.
+     */
+
+    /**
+     * Initialize PrefsEditor global settings store.
+     */
+    fluid.prefs.globalSettingsStore();
+
+    /**
+     * Initialize UI Enhancer for the page.
+     */
+    fluid.pageEnhancer({
+        uiEnhancer: {
+            gradeNames: ["fluid.uiEnhancer.starterEnactors"],
+            tocTemplate: "path/to/TocTemplate"
+        }
+    });
+
+    /**
+     * Initialize Full Page preferences editor
+     */
+    fluid.prefs.fullPreview(".container", {
+        gradeNames: ["fluid.prefs.initialModel.starter"],
+        // Tell PrefsEditor where to find all the templates, relative to this
+        // file
+        terms: {
+            templatePrefix: "path/to/templates",
```

Comment:
Why not supply a fully working example, rather than just some placeholders? That way it would at least be possible to get some assurances that these docs are correct.